### PR TITLE
HATToken to inherit ERC20Votes & ERC20Capped

### PR DESCRIPTION
**Pull request by lirona**

### CODE REVIEW

**Review by jellegerbrandy — DISMISSED** [review]
really very nice :)

**Comment by ben-kaufman** [comment]
Looks good, I think we can replace the use of require with custom errors like the rest of the code. Also could make an onlyGovernance modifier instead of the duplicated require

**Comment by lirona** [comment]
> Looks good, I think we can replace the use of require with custom errors like the rest of the code. Also could make an onlyGovernance modifier instead of the duplicated require

sure, now that i got it to work i'll make an interface and make it all pretty :)

File: contracts/interfaces/IHATToken.sol [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts/governance/utils/IVotes.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/draft-IERC20Permit.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```
Comment (ben-kaufman) [inline contracts/interfaces/IHATToken.sol:6]: Delete this

**Review by shayzluf — APPROVED** [review]
@lirona @ben-kaufman approved

**Pull request merged**